### PR TITLE
Add openSUSE OS projects

### PR DESCRIPTION
**Pull request by nickgerace**
This PR provides a baseline for openSUSE OS-related projects. It does not provide any logos due to due personal lack of knowledge into [openSUSE branding](https://en.opensuse.org/openSUSE:Artwork_brand). However, I believe this PR is a necessary start for adding the projects.

**Note:** all descriptions were taken directly from project sites and repositories.

### CODE REVIEW

File: data/projects/development/zypper.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: Zypper
+repository: https://github.com/openSUSE/zypper
+twitter:
+website: https://en.opensuse.org/Portal:Zypper
```
Comment (nickgerace) [inline data/projects/development/zypper.yaml:5]: Note to reviewers: the page does not load without the `en` prefix, otherwise I would have excluded it.

**Comment by Conan-Kudo** [comment]
@hellcp, you probably will want to look over this.

**Comment by hellcp** [comment]
I would link distros to get.opensuse.org instead of www.o.o and microos.o.o mainly because I am planning on revamping both and those links will no longer be valid.

Kubic project is a project focused on container technologies and not just a distribution, if you want to point at Kubic the distribution it's also gonna be get.o o, but I think Kubic folx need recognition for all they do not just the distro ;)

We have a repository with [official artwork](https://github.com/openSUSE/artwork)  where the logos for various subprojects are stored. libzypp/zypper do not have logos as that seems kinda pointless :P

I also feel that this approach of splitting openSUSE into pieces is just gonna make the whole thing both hard to maintain and exclude many of the things openSUSE does. I would have probably went with a single file for openSUSE instead but you do you.

**Comment by nickgerace** [comment]
@hellcp thank you for the input! I can absolutely put this in one file; that's not a requirement. This PR was merely meant to help start adding openSUSE project(s) since this is a new site.

How about one file for zypper and the other for openSUSE?

**Comment by hellcp** [comment]
Actually one for zypper, one for openSUSE and one for Kubic the project not the distro would probably work best

**Comment by nickgerace** [comment]
Okay great, sounds good! Thanks for the help.

**Comment by oskapt** [comment]
mook already submitted for leap and tumbleweed, which i just merged. if you'd like to undo that and have just opensuse, please rebase and update/consolidate.

**Comment by sysrich** [comment]
> I would link distros to get.opensuse.org instead of [www.o.o](http://www.o.o) and microos.o.o mainly because I am planning on revamping both and those links will no longer be valid.

The like to microos.opensuse.org will remain valid.

**Comment by hellcp** [comment]
> The like to microos.opensuse.org will remain valid.

I see no practical purpose considering the get-o-o revamp that's currently a PR does what both microos-o-o and microos portal do, but better

**Comment by sysrich** [comment]
> > The like to microos.opensuse.org will remain valid.
> 
> I see no practical purpose considering the get-o-o revamp that's currently a PR does what both microos-o-o and microos portal do, but better

the MicroOS website advertises MicroOS as a standalone offering, seperate from all the other openSUSE offerings, to reach a very particular audience that won't go to get-o-o and sift through all the other stuff there.

Please just accept that fact, I won't let you delete the site.

**Comment by hellcp** [comment]
Fair enough, we don't have to advertise it at all on get-o-o, but that doesn't mean it's not the best place to have both info page and download page for it which is gonna be a thing whether you like it or not :shrug:

**Comment by Conan-Kudo** [comment]
There's quite a bit more information on microos-o-o and kubic-o-o than what would be appropriate on get-o-o. They're significantly different and have quite a bit more marketing work to do than your standard openSUSE distributions.

**Comment by oskapt** [comment]
Just checking in - I see all the convo, but I'm waiting for you all to sort it out and collectively say this is ready to merge.

**Comment by nickgerace** [comment]
Thanks for checking in @oskapt. The purpose of this PR was to get an early start into adding openSUSE projects. Since the discussion here has kicked off other PRs that contain more information (like logos), I am closing this PR. Those PRs contain all the projects that this PR would have added.

Thank you for the discussion and subsequent additions to this repo @sysrich @hellcp @Conan-Kudo!

Relevant PRs:
- #44 
- #42
- #41
- #33 
- #2

**Pull request closed without merging**